### DISCUSSION TITLE
Windows 11: UTM stuck on either UEFI Boot, "Guest has not initialized the display (yet).", or purple screen

### DISCUSSION THREAD

**Discussion by l-monninger** [utmapp/UTM#4711] [UTM Help (macOS)]
I'm running into some trouble running Windows 11 on my MacBook pro (Apple Silicon). I have tried both emulating and virtualizing.

I have downloaded all ISOs from uupdump.net.

## Emulating 
I tried emulating several `amd64` Windows ISOs first. I was able to get through to the installer, however, UTM then got stuck displaying a purple screen that never resolved. 

## Virtualizing
I can confirm that I have only tried to virtualize `arm64` packages. If I uncheck UEFI Boot, then I end up getting stuck at... 
>  "Guest has not initialized the display (yet)."

If I check UEFF Boot, I'm stuck at the Shell as described elsewhere. 

I have consulted several forms and YouTube videos on each of these issues, to no avail. Any suggestions?

**Comment by l-monninger** [comment]
Looks like I didn't hit enter enough times/fast enough.

**Comment by FatemaNeemuch** [comment]
I am having the same issue. How did you resolve it?

**Reply by l-monninger** [reply]
I believe I just spammed enter haha.

**Reply by CodeMonkeyCybersecurity** [reply]
spamming enter actually ended up working ahaha
